jormungandr/src/rest/v0/mod.rs

```diff
@@ -222,12 +222,24 @@ pub fn filter(
 
         let vote_plans = warp::path!("plans")
             .and(warp::get())
-            .and(with_context)
+            .and(with_context.clone())
             .and_then(handlers::get_active_vote_plans)
             .boxed();
         root.and(committees.or(vote_plans)).boxed()
     };
 
+    let tokens = {
+        let root = warp::path!("tokens" / ..);
```

Comment:
It looks a bit strange to me that we have a specific route just for tokens where this could be fetched from the account state

Reply:
I have added this new route to fetch directly the specific token amount from the specific account, it seems it should be convenient in some cases.

Reply:
> It looks a bit strange to me that we have a specific route just for tokens where this could be fetched from the account state

And that is not how it is specified. This could (and should) have been just a couple of new fields in the account state structure.